evo/tools/file_interface.py

```diff
@@ -237,6 +237,13 @@ def _get_xyz_quat_from_pose_or_odometry_msg(
     ]
     return xyz, quat
 
+def _get_xyz_quat_from_point_msg(
+        msg) -> typing.Tuple[typing.List[float], typing.List[float]]:
```

Comment:
mypy complains about the typing, either use `typing.List[int]` or change the line below to `quat = [1., 0., 0., 0.]`.

```
evo/tools/file_interface.py:245: error: Incompatible return value type (got "Tuple[List[Any], List[int]]", expected "Tuple[List[float], List[float]]")  [return-value]
```